lighthouse-core/gather/driver.js

```diff
@@ -852,9 +852,14 @@ class Driver {
     return collectUsage;
   }
 
-  blockUrlPatterns(urlPatterns) {
-    const promiseArr = urlPatterns.map(url => this.sendCommand('Network.addBlockedURL', {url}));
-    return Promise.all(promiseArr);
+  /**
+   * This method sets the URL patterns to be blocked and is called at the beginning of each pass.
+   * No "clearing" is done at the end of the pass since the next pass will set to [] if necessary.
+   * @param {!Array<string>} urls
+   * @return {!Promise}
+   */
+  blockUrlPatterns(urls) {
+    return this.sendCommand('Network.setBlockedURLs', {urls});
```

Comment:
unfortunately this command was renamed/added recently..

commit: https://chromium.googlesource.com/chromium/src/+/01c043f5f744262981becd3d45a79981ad3f6b67

m58 (base rev 454471 ) which shipped to stable 2 weeks ago doesn't have this method. It has the add/remove pair instead.

IMO this is optional, and our other use of requestblocking is unused so i think we can ignore the failure. let's catch and exclude errors?

doublecheck this works with a stable m58 build